tests/e2e/scale/test_scale_osds_fill_75%_reboot_workers.py

```diff
@@ -107,4 +107,4 @@ def test_scale_osds_reboot_nodes(self, interface, project_factory, multi_pvc_fac
             nodes.restart_nodes(nodes=[node])
             wait_for_nodes_status()
 
-        assert ceph_health_check(delay=180), "Failed, Ceph health bad after nodes reboot"
+        assert ceph_health_check(delay=180, tries=50), "Failed, Ceph health bad after nodes reboot"
```

Comment:
Same here, AFter a reboot probably the cluster would recover quicker, if not probably it would be a bug, so we need not wait till 50 tries is my opinion.

Reply:
Agreed, I will change the number of tries back to default since we don't need to wait long for cluster to get into a healthy state.